test/parallel/test-fs-realpath.js

```diff
@@ -381,23 +383,13 @@ assertEqualPath(
 //     |   `-- e -> ..
 //     `-- d -> ..
 // realpath(a/b/e/d/a/b/e/d/a) ==> a
-function test_up_multiple(cb) {
+function test_up_multiple(realpath, realpathSync, cb) {
   console.error('test_up_multiple');
   if (skipSymlinks) {
     common.printSkipMessage('symlink test (no privs)');
-    return runNextTest();
-  }
-  function cleanup() {
-    ['a/b',
-     'a'
-    ].forEach(function(folder) {
-      try { fs.rmdirSync(tmp(folder)); } catch (ex) {}
-    });
+    return cb();
   }
-  function setup() {
-    cleanup();
-  }
-  setup();
+  common.refreshTmpDir();
```

Comment:
It gets somewhat lost in the noise but this is a bug fix: the `fs.rmdirSync()` logic didn't work because `a/` contains more than just `a/b`.  It didn't result in actual failures because the test runner cleans it up afterwards.